.gitignore

```diff
@@ -6,4 +6,5 @@
 **/*.prover.*
 **/*.verifier.*
 **universal.srs.*.*
+*.usrs
```

Comment:
I see there are usrs files that are required, we should be pushing those on change and not ignoring them.

Reply:
These are local to the repo, not to `.aleo/resources`, no?

Reply:
For 2^15 they're in the parameters folder